lstchain/datachecks/dl1_checker.py

```diff
@@ -364,8 +351,11 @@ def plot_datacheck(datacheck_filename, out_path=None, muons_dir=None):
     geom = CameraGeometry.from_table(cam_description_table)
     engineering_geom = geom.transform_to(EngineeringCameraFrame())
 
+    '''
```

Comment:
not sure if this is the best way to comment out large blocks of comments. `'''` is usually used for docstrings, but maybe somebody else can comment

Reply:
You're right, I am replacing it with hashes.